holidays/calendars/chinese.py

```diff
@@ -1307,6 +1307,24 @@ class _ChineseLunisolar:
         2053: (FEB, 18),
     }
 
+    WINTER_SOLSTICE_THRESHOLDS: dict[str, dict[str, dict[int, int]]] = {
+        # UTC+7.
+        VIETNAMESE_CALENDAR: {
+            "dec21": {0: 1980, 1: 2017, 2: 2050, 3: 2083},
```

Comment:
It looks like we also use `month_day` format, e.g. https://github.com/vacanza/holidays/blob/b1728bf912418000d300e0a9efa71802ba7e005d/holidays/countries/argentina.py#L278

I suggest keeping it consistent here too.